models/P2PClient/effects.js

```diff
@@ -0,0 +1,90 @@
+const ps = require('peer-star')
+
+module.exports = globalConfig => ({
+  joinStar: joinStar(globalConfig),
+  stop: stop(globalConfig),
+  send: send
+})
+
+function send (state, data, send, done) {
+  if (!state.presenter) {
+    console.log('MAIN not available')
+    return done()
+  }
+  state.presenter.send(JSON.stringify(data))
+  done()
+}
+
+function stop (globalConfig) {
+  return inner
```

Comment:
When declaring the function by `function myFunction () {}` I can return it before declaring without errors. I like to do it this way, because when taking a look at the code you can directly see what it does. For example when you look at the export you can directly see you can call "joinStar", "stop" and "send".
When looking at the "stop" function in line 18 and the (I must admit: weird setup of returning the "inner function"), it directly grants you the insight, that stop is only a wrapper for the actual function, which will be used.